src/web/app/components/question-types/question-edit-answer-form/rank-options-question-edit-answer-form.component.ts

```diff
@@ -55,7 +58,9 @@ export class RankOptionsQuestionEditAnswerFormComponent
    * Assigns a Rank to the option specified by index.
    */
   triggerResponse(index: number, event: any): void {
-    this.responseDetails.answers[index] = event;
+    const copyAnswers: number[] = this.responseDetails.answers.slice();
```

Comment:
```suggestion
    const newAnswers: number[] = this.responseDetails.answers.slice();
```

Can we call it as newAnswer or something instead of copyAnswers which does not fit the context.